RDF-Pipeline/t/tests/0056_Set_Perl_Lib_Env_Var/expected-files/node/perlenv

```diff
@@ -5,27 +5,33 @@ use strict;
 # Verify that the Perl lib and other env vars are set properly 
 # to include RDF Pipeline libs.
 
+
 # The perl lib path:
+print "\nVerifying \@INC contains RDF-Pipeline path\n\n";
 my $inc = join(" ", @INC) || "";
-print "\@INC=$inc\n\n";
-$inc =~ m/Pipeline/ or die "Perl \@INC is not set properly: $inc\n";
+$inc =~ m/RDF-Pipeline\/lib/ or die "Perl INC is not set properly: $inc\n";
 
+print "Verifying PERL5LIB contains RDF-Pipeline/lib path\n\n";
 my $lib = $ENV{PERL5LIB} || "";
-print "PERL5LIB=$lib\n\n";
 $lib || die "\$PERL5LIB is not set!\n";
+$lib =~ m/RDF-Pipeline\/lib/ or die "Perl PERL5LIB is not set properly: $inc\n";
```

Comment:
Oops, this comment should actually be applied to setup-files/node/perlenv, and then automatically propagated to the expected-files when you run accept-test.perl.